### DISCUSSION TITLE
How to install macOS PowerPC on UTM 4.4.4 on Sonoma in a way it actually works?

### DISCUSSION THREAD

**Discussion by barracuda156** [utmapp/UTM#5904] [macOS]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
10.5.8 PowerPC does not boot. Tried both setting it up from scratch and using a pre-made config from https://github.com/adespoton/utmconfigs

Issue I face looks identical to this: https://github.com/adespoton/utmconfigs/issues/6
I get the same yellow screen instead of OS booting.

**Configuration**  
* UTM Version: 4.4.4
* macOS Version: 14.1.1
* Mac Chip (Intel, M1, ...): M1

**Comment by barracuda156** [comment]
@osy Is it known to be working otherwise, and just fails for me? (Which is certainly possible: I am no expert in setting up VMs at all.)

**Reply by LukasMFR** [reply]
Here is my situation regarding Mac OS in UTM (note that I have Sonoma and I have an M3 Pro MacBook Pro) :
- Mac OS X Lion 10.7 is working fine, installed and booting to the OS (but quite slow)
- Mac OS X Jaguar 10.2 : the install CD is booting but I just can't get the OS to boot
- Mac OS X Leopard 10.5 : I just can't get the DVD to boot (but I am looking at other DVDs)


**Note that I have :**
- MacBook Pro M3 Pro
- UTM version 4.4.2 (90) (I read somewhere that this version worked better, in fact I couldn't make Lion work on the latest version)
- macOS Sonoma 14.2.1 (23C71)
- I always used the premade configs from GitHub

**Reply by kingbuzzman** [reply]
Did you get this to work? I'm trying this out now, and having the same issue.

**Reply by barracuda156** [reply]
> Did you get this to work? I'm trying this out now, and having the same issue.

I did not try for quite some time, but from your situation it looks like everything is still broken…

**Reply by Woody123123** [reply]
https://winworldpc.com/product/mac-os-x/103
this may help
